catroid/src/main/java/org/catrobat/catroid/utils/StatusBarNotificationManager.java

```diff
@@ -49,8 +51,9 @@
 	public static final String ACTION_CANCEL_UPLOAD = "cancel_upload";
 
 	private static final StatusBarNotificationManager INSTANCE = new StatusBarNotificationManager();
+	private static AtomicInteger uniqueNotificationIdCounter = new AtomicInteger(0);
```

Comment:
The `StatusBarNotificationManager` is a Singleton. Let's us it that way. Remove static here (adapt code), readd final.